dune

```diff
@@ -25,7 +25,11 @@
   (source_tree theories)
   (source_tree plugins)
   (source_tree user-contrib))
- (action (with-stdout-to .vfiles.d (bash "%{bin:coqdep} -dyndep both -noglob -boot `find theories plugins user-contrib -type f -name *.v`"))))
+ (action
+  (with-stdout-to .vfiles.d
+   (bash "%{bin:coqdep} -dyndep both -noglob -boot -R theories Coq -R plugins Coq -Q user-contrib/Ltac2 Ltac2 -I user-contrib/Ltac2 \
+         -I plugins/btauto -I plugins/cc -I plugins/derive -I plugins/extraction -I plugins/firstorder -I plugins/fourier -I plugins/funind -I plugins/ltac -I plugins/micromega -I plugins/nsatz -I plugins/omega -I plugins/rtauto -I plugins/setoid_ring -I plugins/ssr -I plugins/ssrmatching -I plugins/syntax \
```

Comment:
Wouldn't it be better to use something like `$(find plugins/ -maxdepth 1 -mindepth 1 -printf '-I %p')` instead of explicitly listing plugins (which means we have to keep the list updated)?

Reply:
Good point; I didn't care so much as this target is hopefully going away soon anyways.

Reply:
Updated.